bokeh/_json_encoder.py

```diff
@@ -79,4 +79,4 @@ def default(self, obj):
             return self.transform_python_types(obj)
 
 def serialize_json(obj, encoder=BokehJSONEncoder, **kwargs):
-    return json.dumps(obj, cls=encoder, allow_nan=False, **kwargs)
+    return json.dumps(obj, cls=encoder, allow_nan=True, **kwargs)
```

Comment:
So here is my thinking on the reviver: I'd like to move to binary encoding of arrays for both server cases and even really for static cases. Then we can see if there is a way to eliminate "single" nan/infs in properties, or if we can't at least only the small lightweight models have to go through the reviver.